external-crates/move/crates/move-cli/src/base/new.rs

```diff
@@ -58,27 +58,60 @@ impl New {
         create_dir_all(path.join(SourcePackageLayout::Sources.path()))?;
         let mut w = std::fs::File::create(path.join(SourcePackageLayout::Manifest.path()))?;
         writeln!(
-            &mut w,
+            w,
             "[package]
 name = \"{name}\"
 
+# version = \"0.1.1\"  # semver \"maj.min.patch\"
+# license = \"\"       # e.g., \"MIT\", \"GPL\", \"Apache 2.0\"
+# authors = [\"...\"]  # e.g., [\"Joe Smith (joesmith@noemail.com)\", \"John Snow (johnsnow@noemail.com)\"]
+
 [dependencies]"
         )?;
         for (dep_name, dep_val) in deps {
             writeln!(w, "{dep_name} = {dep_val}")?;
         }
 
-        writeln!(
-            w,
-            "
-[addresses]"
-        )?;
+        writeln!(w, "
+# For remote import, use the `{{ git = \"...\", subdir = \"...\", rev = \"...\" }}`.
+# Revision can be a branch, a tag, and a commit hash.
+# MyRemotePackage = {{ git = \"https://some.remote/host.git\", subdir = \"remote/path\", rev = \"main\" }}
+
+# For local dependencies use `local = path`. Path is relative to the package root
+# Local = {{ local = \"../path/to\" }}
+
+# To resolve a version conflict and force a specific version for dependency
+# override use `override = true`
+# Override = {{ override = true, local = \"../conflicting/version\" }}
```

Comment:
Curious if we can expand on the use of overrides in the docs and link to it from here (I recently had questions about that myself).

May also be useful to link to docs for other Move.toml contents here (though only as long as we have a high confidence links don't break)

Reply:
We cannot link to Sui docs, because this is not `sui` and a more generic "move cli".